<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>699</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>798</width>
    <height>699</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>798</width>
    <height>699</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>798</width>
    <height>699</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>10</y>
      <width>391</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_archives">
     <item>
      <widget class="QLabel" name="label_archives">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Archives in Repository:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_archive">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>540</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_repository">
     <item>
      <widget class="QLabel" name="label_repo_details">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Repository Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_repo_original_size">
       <property name="text">
        <string>Original Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_repo_compressed_size">
       <property name="text">
        <string>Compressed Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_repo_deduplicated_size">
       <property name="text">
        <string>Deduplicated Size:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>10</y>
      <width>391</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_files">
     <item>
      <widget class="QLabel" name="label_filetree">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Choose folder to backup:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeview_files"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_backup"/>
   <addaction name="action_restore"/>
   <addaction name="action_mount"/>
   <addaction name="action_delete"/>
   <addaction name="action_settings"/>
   <addaction name="action_help"/>
  </widget>
  <action name="action_backup">
   <property name="icon">
    <iconset>
     <normaloff>../icons/save.svg</normaloff>../icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="toolTip">
    <string>Create an archive of the selected folder.</string>
   </property>
  </action>
  <action name="action_restore">
   <property name="icon">
    <iconset>
     <normaloff>../icons/rotate-ccw.svg</normaloff>../icons/rotate-ccw.svg</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore a complete archive to a location of your choice.</string>
   </property>
  </action>
  <action name="action_mount">
   <property name="icon">
    <iconset>
     <normaloff>../icons/hard-drive.svg</normaloff>../icons/hard-drive.svg</iconset>
   </property>
   <property name="text">
    <string>Mount</string>
   </property>
   <property name="toolTip">
    <string>Mount an archive as a volume to restore single files.</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset>
     <normaloff>../icons/delete.svg</normaloff>../icons/delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete the selected archive.</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset>
     <normaloff>../icons/help-circle.svg</normaloff>../icons/help-circle.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Open the documentation for Borg-Qt.</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="icon">
    <iconset>
     <normaloff>../icons/settings.svg</normaloff>../icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open the settings window to configure Borg-Qt.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
